ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/logging/RDXLeRobotDatasetCreator.java

```diff
@@ -140,6 +149,9 @@ private void renderImGuiWidgets()
          ImGui.text("Current task name:");
          ImGuiTools.inputTextMultiline(labels.getHidden("taskName"), imTaskName);
 
+         ImGui.setNextItemWidth(100.0f);
+         ImGui.inputInt(labels.get("Auto Scrub Task ID Filter"), imTaskID);
```

Comment:
This improvement to the episode generator allows you specify which task ID to auto scrub for.